data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,450 @@ mission "Unfettered returning home"
+			choice
+				`	(Turn around.)`
+			`	The waves become agitated and wind rushes past you as a shape surrounded by fumes emerges out of the water, as if an ancient sea monster was rising from the depths.`
+			`	But it is no monster. It is a ship, and its weapons are aimed straight at you!`
+			label "try again"
+			scene "scene/emerging ladybug"
+			choice
+				`	(Run!)`
+				`	(Wait.)`
+					goto wait
+				`	(Try to contact my fleet for help.)`
+					to display
+						"total ships" > 1
+				`	(Shoot it!)`
+					goto foolish
+			`	As you try to react, the Hai closest to you firmly grabs your arm and shouts through the noise. "If this ship wanted to kill you, <last>, you would already be dead!"`
+			label wait
+			`	You can barely hear anything over the sound of the engines as the ship proceeds to land on the beach. Suddenly, all is calm. They turn to you, one more at ease than the other. "This ship means you no harm."`
+			choice
+				`	(Say nothing.)`
+					goto explain
+				`	"Why didn't you say anything before that happened?"`
+			`	The second one replies, "We could have, but where's the fun in that? We wanted to see how you would react.`
+			label explain
+			`	"This ship is our guarantee that nothing goes wrong, should you attempt something against us. After all, we are still not sure if we can trust you."`
+			`	The two Hai then take you to a nearby cave. Crystals embedded in the walls cast a faint blue glow as you follow them, until you approach a large chamber where sunlight breaks through cracks in the walls, revealing another ship. The first Hai then speaks again. "That ship outside is not for sale. This one, however, is a mighty relic of the past, intended for war from the times our empire used to control the stars - an Anomalocaris!"`
+			scene "scene/cave scene"
+			`	The second one now takes a step toward you to speak, almost interrupting the first. "We have a proposition for you. Should you agree to bring much needed supplies to our brethren, the Unfettered Hai, we will sell you this ship for the modest price of ten million credits - including two weapons belonging to the savage Korath!" The first one seems surprised by this and tries to intervene, but the second resumes before they can speak up.`
+			`	"These legendary weapons can interfere with nearly any engine system! Worry not, you'll be able to recuperate the cost of this ship with ease by assisting us. A load of medical supplies is needed on <destination>, and we lack the time and personnel to bring them ourselves on such short notice.`
+			`	"Of course, the ship is aged, and you should not expect the best price in a shipyard. Nevertheless, it is a fine vessel, capable of fighting, freighting, and mining. So, what do you say, <last>? This is the opportunity of a lifetime; they simply don't make ships like this anymore."`
+			label choice
+			choice
+				`	"I will help you."`
+					goto take
+				`	"If you lack crew, who was piloting the ship from the ocean?"`
```

Comment:
```suggestion
				`	"If you lack crew, who was piloting the other ship?"`
```
I think keeping this short and sweet is fine; it's not clear that it was from an ocean or lake, etc.. Also below, the option now mentions under water, and with the above, it's pretty clear this means the ship from before, since we're just staring at this new Anomalocaris.